projects/schematics/src/migrations/4_0/constructor-deprecations/data/configurator-attribute-numeric-input-field.component.migration.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  CONFIGURATOR_ATTRIBUTE_NUMERIC_INPUT_FIELD_COMPONENT,
+  CONFIGURATOR_UI_SETTINGS_CONFIG,
+  SPARTACUS_PRODUCT_CONFIGURATOR_RULEBASED,
+} from '../../../../shared/constants';
+import { ConstructorDeprecation } from '../../../../shared/utils/file-utils';
+
+export const CONFIGURATOR_ATTRIBUTE_NUMERIC_INPUT_FIELD_COMPONENT_MIGRATION: ConstructorDeprecation = {
+  // feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.ts
+  class: CONFIGURATOR_ATTRIBUTE_NUMERIC_INPUT_FIELD_COMPONENT,
+  importPath: SPARTACUS_PRODUCT_CONFIGURATOR_RULEBASED,
+  deprecatedParams: [],
```

Comment:
Shouldn't we put in `deprecatedParams` array the reference to the service `ConfiguratorAttributeNumericInputFieldService`?